index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio | M Nisar Ali</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href='https://fonts.googleapis.com/css?family=Playfair Display' rel='stylesheet'>
+    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Poppins" />
+</head>
+
+<body>
+    <header>
+        <nav class="nav">
+            <a href="#">
+                <img src="./Images/ic_menu.svg" alt="Navigation menu">
+            </a>
+        </nav>
+    </header>
+    <main>
+        <section class="headline">
+            <h1>Muhammad Nisar Ali</h1>
+            <div class="intro">
+                <p>
+                    Hello! I am a software developer! I can help you build a product, feature
+                    or website. Take a look of my works. If you like what you see and have a
+                    project you need coded, don't hesitate and contact me.
+                </p>
+            </div>
+            <a class="collab-button" href="#">
+                <button class="button" type="button">Start collaboration</button>
+            </a>
```

Comment:
[OPTIONAL] 👍🏽 
- I think it will be better to use either an `<a>` tag alone or a `<button>`, instead of having to use both at the same time.

 I think it would look better this way 👉🏽 `<button class="button collab-button">Start collaboration</button>`

I have also removed the `type="button"` as I believe this is often used when you need to submit a form. But in our case, the "start collaboration" is not a form.  🌸 🌸

Reply:
Hi @Favourezeugwa,
Thank you so much for your suggestion and review. 🌸